demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py

```diff
@@ -308,3 +309,28 @@ def validate_duplicate_ids(
         (item.get("content_item"), item.get("duplicate_content_items"))
         for item in run_query(tx, query)
     ]
+
+
+def validate_test_playbook_in_use(
+    tx: Transaction, test_playbook_ids: List[str], tests_skipped: List[str]
+) -> List[graph.Node]:
+    query = f"""
+MATCH (tp:TestPlaybook)
+WHERE tp.object_id IN {test_playbook_ids}
+AND NOT EXISTS {{ MATCH ()-[:TESTED_BY]->(tp) }}
+AND tp.deprecated = false
+AND NOT (tp.object_id IN {tests_skipped})
+MATCH (tp)-[:IN_PACK]->(p:Pack)
+WHERE p.support = "xsoar"
+AND p.deprecated = false
+RETURN collect(tp) AS content_items
+"""
+    # when there a test playbooks that not in use, the query return a list with one item
+    return first(
```

Comment:
```suggestion
    return one(
```
this also fails if there's `0` or `>1` values

Reply:
When all the test playbooks are in use, the query returns an empty list. I don't want the code to fail, so we have two options:
1. A list with one value - indicating that there is a test playbook not in use.
2. An empty list - indicating that all the test playbooks are in use.